clients/drcachesim/simulator/tlb_simulator.cpp

```diff
@@ -283,17 +283,14 @@ tlb_simulator_t::print_results()
 tlb_t *
 tlb_simulator_t::create_tlb(std::string policy)
 {
-    // XXX: how to implement different replacement policies?
```

Comment:
Please keep a comment here with `XXX i#7287: ...` on refactoring replacement policies.  Otherwise, your PR here looks like you think your tlb_lfu_t, etc. are the final solution, when we think they are a temporary solution that should be replaced by the #7287 solution.